ppr-ui/src/views/ConfirmMHRSearch.vue

```diff
@@ -164,6 +164,8 @@ export default class ConfirmDischarge extends Vue {
   private submitting = false
   private validFolio = true
   private staffPaymentValid = false
+  private validating = false
+  private curOption = StaffPaymentOptions.NONE
```

Comment:
Took me a second to understand the property name 😆, is it possible to use a full name or a name a little more concise to it's purpose?

ie `currentOption` or `currentPaymentOption` or `paymentOption` etc etc and so on and so on 😁